content/en/docs/concepts/services-networking/service.md

```diff
@@ -1422,30 +1387,12 @@ VIP, their traffic is automatically transported to an appropriate endpoint.
 The environment variables and DNS for Services are actually populated in
 terms of the Service's virtual IP address (and port).
 
-kube-proxy supports three proxy modes&mdash;userspace, iptables and IPVS&mdash;which
-each operate slightly differently.
-
-#### Userspace
-
-As an example, consider the image processing application described above.
-When the backend Service is created, the Kubernetes master assigns a virtual
-IP address, for example 10.0.0.1.  Assuming the Service port is 1234, the
-Service is observed by all of the kube-proxy instances in the cluster.
-When a proxy sees a new Service, it opens a new random port, establishes an
-iptables redirect from the virtual IP address to this new port, and starts accepting
-connections on it.
-
-When a client connects to the Service's virtual IP address, the iptables
-rule kicks in, and redirects the packets to the proxy's own port.
-The "Service proxy" chooses a backend, and starts proxying traffic from the client to the backend.
-
-This means that Service owners can choose any port they want without risk of
-collision.  Clients can connect to an IP and port, without being aware
-of which Pods they are actually accessing.
+kube-proxy supports two proxy modes&mdash;iptables and IPVS&mdash;which
+operate slightly differently.
```

Comment:
There is also a `kernelspace` mode - see https://github.com/kubernetes/website/issues/38244

Reply:
This PR necessarily conflicts with that one (that PR modifies the text immediately before the `### User space proxy mode` heading which this PR removes), so one of us will have to rebase